### ISSUE TITLE
Cuts lead to incorrect results

### ISSUE THREAD

**Issue by darrylmelander** [coin-or/Cbc#337]
Here's a pretty simple model that gives incorrect results with "cuts on" but correct results with "cuts off".  I'm using cbc-2.10.5 command line.  Here the LP file:

```
Minimize
596111.2725 b0 + 
 135908.703 b1 + 
 1000000 g0 + 
 12000000 g1 + 
 3900000 g2 + 
 800000 g3 + 
 6000000 g4 + 
 0
Subject To
1192.222545 b0 
 + 271.817406 b1 
 + 1000 g0 
 + 18000 g1 
 + 5000 g2 
 + 500 g3 
 + 8000 g4 
 >= 1527.039951
Binary
b0 
 b1 
General
g0 
 g1 
 g2 
 g3 
 g4 
End
```

The correct minimum objective value is 1396111.2725.  If you run this in cbc without first turning off cuts it reports 1596111.2725.

**Comment by jjhforrest** [comment]
Seems Okay with master - can reproduce with 2.10.5.

Will look at it

John Forrest

**Comment by tkralphs** [comment]
This should be fixed now by coin-or/Cgl@5873be4. For now, to get the fix, you will need to get Cbc stable/2.10. You can either build from source with
```
coinbrew build Cbc@2.10
```
or get binaries from [bintray](https://bintray.com/coin-or/download/Cbc/2.10).

The fix will be part of release 2.10.6 when that comes out.

**Comment by darrylmelander** [comment]
> This should be fixed now 

Thank you
